reolinkapi/mixins/network.py

```diff
@@ -16,17 +18,46 @@ def set_net_port(self, http_port: float = 80, https_port: float = 443, media_por
         :type http_port: int
         :return: bool
         """
+        if http_enable:
+            http_enable = 1
+        else:
+            http_enable = 0
+        if https_enable:
+            https_enable = 1
+        else:
+            https_enable = 0
+        if onvif_enable:
+            onvif_enable = 1
+        else:
+            onvif_enable = 0
+        if rtmp_enable:
+            rtmp_enable = 1
+        else:
+            rtmp_enable = 0
+        if rtsp_enable:
+            rtsp_enable = 1
+        else:
+            rtsp_enable = 0
+
         body = [{"cmd": "SetNetPort", "action": 0, "param": {"NetPort": {
+            "httpEnable": http_enable,
             "httpPort": http_port,
+            "httpsEnable": https_enable,
             "httpsPort": https_port,
             "mediaPort": media_port,
+            "onvifEnable": onvif_enable,
             "onvifPort": onvif_port,
+            "rtmpEnable": rtmp_enable,
             "rtmpPort": rtmp_port,
+            "rtspEnable": rtsp_enable,
             "rtspPort": rtsp_port
```

Comment:
We could rather use `int()` to convert the boolean instead of writing an if statement to map the boolean values to an integer.

```
~ ⟩ python3
Python 3.11.2 (main, Feb  7 2023, 13:52:42) [GCC 11.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> print(int(True))
1
>>> print(int(False))
0
```

Reply:
probably don't even need to convert from `bool` to `int` so the variable can be passed along directly